### PR TITLE
Remove outdated links from `Performance points`

### PR DESCRIPTION
**Pull request by gitMiloTilo**
Remove all mentions of the link "https://osu.ppy.sh/home/changelog?category=pp" since it doesn't lead to anything or otherwise inaccessible.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Performance_points/en.md [hunk 1]
```diff
@@ -26,7 +24,7 @@ On January 16, 2021, changes were made to the ppv2 system that aimed to more acc
 - Adjust the rate of pp loss from misses to be more forgiving on longer maps with higher combo
 - Punish lower [accuracy](/wiki/Gameplay/Accuracy) plays with less pp gain
 
-ppv2 is currently in active service, with live updates published to its [changelog](https://osu.ppy.sh/p/changelog?category=pp).
+ppv2 is currently in active service, with live updates published by the [PP Committee](https://osu.ppy.sh/wiki/en/People/Performance_Points_Committee)
```
Comment (agatemosu) [inline wiki/Performance_points/en.md:27]:
```suggestion
ppv2 is currently in active service, with live updates published by the [PP Committee](/wiki/en/People/Performance_Points_Committee)
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Performance_points/en.md [hunk 2]
```diff
@@ -26,7 +24,7 @@ On January 16, 2021, changes were made to the ppv2 system that aimed to more acc
 - Adjust the rate of pp loss from misses to be more forgiving on longer maps with higher combo
 - Punish lower [accuracy](/wiki/Gameplay/Accuracy) plays with less pp gain
 
-ppv2 is currently in active service, with live updates published to its [changelog](https://osu.ppy.sh/p/changelog?category=pp).
+ppv2 is currently in active service, with live updates published by the [PP Committee](/wiki/en/People/Performance_Points_Committee)
```
Comment (agatemosu) [inline wiki/Performance_points/en.md:27]:
```suggestion
ppv2 is currently in active service, with live updates published by the [PP Committee](/wiki/People/Performance_Points_Committee)
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Performance_points/en.md (hunk 2 above)
Comment (gitMiloTilo) [reply wiki/Performance_points/en.md]: sorry I almost missed this suggestion lol

**Comment by gitMiloTilo** [comment]
drafted for minor fixes to the PR

**Comment by gitMiloTilo** [comment]
Added
outdated_translation and outdated_since tags to non-english articles.

**Comment by gitMiloTilo** [comment]
actually might reverse since it's pretty minor

**Comment by agatemosu** [comment]
it's already done

**Comment by agatemosu** [comment]
it's already done

**Comment by TicClick** [comment]
an important point before I merge this: please focus on the changes being made and batch them where possible (I have combined your 12 commits into 2, but this required history rewrite, hence the force push)

**Comment by TicClick** [comment]
(also, the links are still important for historical purposes -- I saved them into https://github.com/ppy/osu-wiki/issues/5898 with the `?category=pp` link wrapped in the wayback machine)

**Review by TicClick — APPROVED** [review]

**Pull request merged**